docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: "3.7"
+services:
+  nginx:
+    build:
+      context: infra/nginx
+      dockerfile: Dockerfile
+    container_name: nginx
+    ports:
+      - target: 80
+        published: 8080
+        protocol: tcp
+        mode: host
+    links:
+      - vvgo
+      - minio
+    networks:
+      - frontend
+      - backend
+
+  vvgo:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: vvgo
+    environment:
+      MINIO_ENDPOINT: minio:9000
+      MINIO_ACCESS_KEY_ID: minioadmin
+      MINIO_SECRET_ACCESS_KEY: minioadmin
```

Comment:
Is this for Test or Prod? We should probably use some variety of Secret for this.

Reply:
Ya this is still testing. We can expose secrets somewhere.